tests/main/snap-repair/uc16.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -ex
+
+# show what mode we are in
+echo "$SNAP_SYSTEM_MODE"
+
+# mark us as done using the snap-repair command manually
+/usr/lib/snapd/snap-repair "done"
```

Comment:
why the quotes around "done"  ?

Reply:
otherwise shellcheck complains about SC2010 in my editor, though for some reason spread-shellcheck doesn't complain :thinking: Do you feel strongly about removing the quotes?

Reply:
no